### ISSUE TITLE
VariablesInputDevice incorrectly increases

### ISSUE THREAD

**Issue by wutu** [pimatic/pimatic#968]
If I have this setup:
~~~
    {
      "variable": "23",
      "type": "number",
      "min": 21,
      "max": 26,
      "step": 0.1,
      "id": "fishtank-temp-set",
      "name": "Fishtank temp set",
      "class": "VariableInputDevice"
    }
~~~
When increasing the value I get for example:
```22.000000000000004```
In reducing everything is OK.

**Comment by mwittig** [comment]
I can reproduce the problem. Moreover, reducing the value sometimes decrements by 0.2 instead of 0.1

**Comment by wutu** [comment]
Hi,  I have same problem with InputDevice.

**Comment by mwittig** [comment]
Yes, it a bug in the UI part of the input thingy which is part of pimatic-mobile-frontend

**Comment by koffienl** [comment]
Came acros the same bug.
min : 0.1
max : 1
step : 0.1

It will increase to strange steps like going from 0.2 to 0.30000000000000004
Furthermore, it seems to be impossible to increase beyond 0.5 when the max is set to be 1 ?

**Comment by mwittig** [comment]
> Is there focus on this bug?

Unfortunately, there is no focus on this right now. I had investigated the matter to be a problem with the jQM version we're using. Unfortunately, I am not an expert on this. I'll make a call for help on the Forum.

https://forum.pimatic.org/topic/4181/call-for-help-frontend

**Comment by koffienl** [comment]
Thanks :)
Unfortunately this kind of development is beyond my knowledge.

**Comment by betz0r** [comment]
I will dig into this as i encountered an issue myself with VariableInputDevice,the + and - buttons won't work at all in my installation on desktop but work on mobile.
